src/buildercore/terraform.py

```diff
@@ -196,6 +196,7 @@ def render_fastly(context):
             b['healthcheck'] = 'default'
 
     _render_fastly_errors(context, template, vcl_templated_snippets)
+    _render_fastly_journal_submit(context, template, vcl_templated_snippets)
```

Comment:
similar to the `xpub` configuration, this module should not know about specific project hence this can be the generic `_render_fastly_vcl_templates()`